ps_distributionapiclient.php

```diff
@@ -97,4 +105,100 @@ private function getDistributionApi(): DistributionApi
 
         return $distributionApi;
     }
+
+    /**
+     * @return string
+     */
+    public function getContent()
+    {
+        $html = $this->renderForm();
+
+        if (Tools::isSubmit(static::SUBMIT_NAME)) {
+            Configuration::updateValue(
+                static::SEND_SHOP_URL,
+                Tools::getValue(static::SEND_SHOP_URL)
+            );
+
+            /** @var Link $link */
+            $link = $this->context->link;
+            Tools::redirectAdmin($link->getAdminLink('AdminModules') . '&configure=' . $this->name . '&conf=6');
+        }
+
+        return $html;
```

Comment:
Wouldn't it be great to do it the Symfony way?

Reply:
I can do it Symfony-way, but in the case of one checkbox I don't see any added value.

@eternoendless what do you think?

Reply:
Well, to me it's more about being a example for the community. If we're not using Symfony, we can't expect external developers of modules to do it

Reply:
Of course, but we need to deliver this ASAP